### PR TITLE
Fixing issues from weekly CATS report

### PR DESCRIPTION
**Pull request by mairaw**
Fixing some of the issues showing up on the report.

### CODE REVIEW

File: docs/visual-basic/language-reference/functions/string-functions.md [hunk 1]
```diff
@@ -86,4 +86,4 @@ This example shows various uses of the `Format` function to format values using
 - [Keywords](../../../visual-basic/language-reference/keywords/index.md)
 - [Visual Basic Runtime Library Members](../../../visual-basic/language-reference/runtime-library-members.md)
 - [String Manipulation Summary](../../../visual-basic/language-reference/keywords/string-manipulation-summary.md)
-- [System.String class methods]<xref:System.String#methods?displayProperty=nameWithType>
+- [System.String class methods](/dotnet/api/system.string#methods)
```
Comment (tdykstra) [inline docs/visual-basic/language-reference/functions/string-functions.md]: Would xref without ?displayProperty=nameWithType work, i.e. `[System.String class methods](xref:System.String#methods)`?

**Review by tdykstra — APPROVED** [review]
One question.

File: docs/visual-basic/language-reference/functions/string-functions.md (hunk 1 above)
Comment (mairaw) [reply docs/visual-basic/language-reference/functions/string-functions.md]: I don't think so but I'll test on a separate branch.

Comment (mairaw) [reply docs/visual-basic/language-reference/functions/string-functions.md]: Yep, confirmed. It doesn't.

**Pull request merged**